src/util/animation/easing.ts

```diff
@@ -1,10 +1,16 @@
 /**
  * Easing functions
- * @see {@link http://gizma.com/easing/ Easing Equations by Robert Penner}
+ * See <a href="http://gizma.com/easing/">Easing Equations by Robert Penner</a>
```

Comment:
`@see {@link http://gizma.com/easing/ Easing Equations by Robert Penner}` is better